### PR TITLE
[Scale FS] Creating max subvolumes with 1 gb each

### PR DESCRIPTION
**Pull request by AmarnatReddy**
[Scale FS] Creating max subvolumes with 1 gb each
Test Script Flow :
    1. We will get the total available space.
    2. Based on total Available space we create a number of volumes with 1 GB each
    3. Verify there is no error after that

Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-E1YZBC/

Signed-off-by: Amarnath K <amk@amk.remote.csb>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by hkadam134 — DISMISSED** [review]
Looks Goot to me

File: tests/cephfs/cephfs_scale/max_subvolumes.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_{idx}/"
+                fs_util.fuse_mount([clients[0]], fuse_mounting_dir_1)
+                clients[0].exec_command(
+                    sudo=True,
+                    cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation create --threads 10 "
+                    f"--file-size 1024 "
+                    f"--files 1024 --files-per-dir 10 --dirs-per-dir 1 --top "
+                    f"{fuse_mounting_dir_1}",
+                    long_running=True,
+                )
```
Comment (yogesh-mane) [inline tests/cephfs/cephfs_scale/max_subvolumes.py:87]: Hey @AmarnatReddy , could you check if this small file is creating 1 GB data. In logs I'm seeing very less data in 'total_used_bytes' in "ceph df" & 'used'  in "ceph fs status"

Comment (AmarnatReddy) [reply tests/cephfs/cephfs_scale/max_subvolumes.py:97]: latest logs for this Yogesh. I made a few changes to that
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-3P0UB4/

Can you review it again

**Review by yogesh-mane — APPROVED** [review]
LGTM

**Pull request merged**